rubicon/objc/objc.py

```diff
@@ -948,8 +948,7 @@ def setter(_self, new):
         def _objc_getter(objc_self, objc_cmd, *args):
```

Comment:
Can the `*args` here and on `_objc_setter` be replaced with zero/one explicit parameters as well?

Reply:
Turns out - yes! And the method is a lot cleaner for it.